example/lib_api/test.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can we add shebang with python version using "env"?

`#!/usr/bin/env python3`

Reply:
Added.